spaces/domains.json

```
@@ -381,5 +381,6 @@
   "vote.mgh.dao": "metagamehub.eth",
   "gov.maidcoin.org": "maidcoin.eth",
   "vote.cyberkongz.com": "cyberkongz.eth",
+  "dao.baka.cool": "koumakan.eth"
   "governance.yfdai.finance": "yfdaifinance.eth"
```

A comma is missing at the end of the line above your sub-domain.
```suggestion
  "dao.baka.cool": "koumakan.eth",
  "governance.yfdai.finance": "yfdaifinance.eth"
```